### DISCUSSION TITLE
I got the front page on the server but no working site just "Hello world! This is HTML5 Boilerplate."

### DISCUSSION THREAD

**Discussion by AusGirl** [h5bp/html5-boilerplate#2438] [General]
I got the front page on the server but no working site just "Hello world! This is HTML5 Boilerplate." http://localhost:1234/ 

So, where is the website? what should I do to view it? I'm on VSCode

**Comment by mattjjohnson007** [comment]
don't suppose you ever patched this issue as i am running into the same problem..

**Reply by roblarsen** [reply]
Hi @mattjjohnson007 There's nothing to patch. When you first run the project, that's all you should see. Take a look at the source code of the index.html. that's all there is to it. 

```
  <!-- Add your site or application content here -->
  <p>Hello world! This is HTML5 Boilerplate.</p>

```

I would be curious what you _expect_ to see, however, because we might be able to fix the documentation.
